lang/js/lib/schemas.js

```diff
@@ -1789,8 +1790,9 @@ LogicalType.prototype._write = function (tap, any) {
 };
 
 LogicalType.prototype._check = function (any, cb) {
+  var val;
```

Comment:
Not a variable name that I would pick, but this is unrelated to applying the lint.